src/Mod/TechDraw/Gui/DlgPrefsTechDrawGeneral.ui

```diff
@@ -229,8 +229,12 @@ for ProjectionGroups</string>
             <height>0</height>
            </size>
           </property>
+          <property name="toolTip">
+           <string>* this font is also used for dimensions
+   Changes have no effect on existing dimensions.</string>
```

Comment:
Alternate wording and capitalization suggestion:
```suggestion
           <string>* Also used for dimensions, but changes here only affect new dimensions.</string>
```

Reply:
> Alternate wording and capitalization suggestion:

Thanks but I prefer here my wording.